api/src/main/java/shop/ink3/api/payment/paymentUtil/processor/impl/TossPaymentProcessor.java

```diff
@@ -0,0 +1,41 @@
+package shop.ink3.api.payment.paymentUtil.processor.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import shop.ink3.api.payment.dto.PaymentConfirmRequest;
+import shop.ink3.api.payment.paymentUtil.processor.PaymentProcessor;
+
+import java.util.Map;
+import java.util.Base64;
+import shop.ink3.api.payment.paymentUtil.client.PaymentClient;
+
+@RequiredArgsConstructor
+@Component("TOSS-PROCESSOR")
+public class TossPaymentProcessor implements PaymentProcessor {
+
+    @Value("${payment.toss.secret_key}")
+    private String SECRET_KEY;
+    private final PaymentClient tossPaymentClient;
+    private final String PAYMENT_KEY = "paymentKey";
+    private final String PAYMENT_AMOUNT = "amount";
+    private final String PAYMENT_ORDER_ID = "orderId";
```

Comment:
static 키워드까지 기입해 상수로 명시해주는건 어떻게 생각하시나요?!

```suggestion
    private static final String PAYMENT_KEY = "paymentKey";
    private static final String PAYMENT_AMOUNT = "amount";
    private static final String PAYMENT_ORDER_ID = "orderId";
```